niworkflows/interfaces/registration.py

```diff
@@ -243,6 +245,8 @@ def _post_run_hook(self, runtime):
 
         self._fixed_image = target_file
         self._moving_image = res.outputs.transformed_file
+        if mri_dir is not None:
+            self._contour = os.path.join(mri_dir, 'ribbon.mgz')
```

Comment:
Do our reports work with mgz files? Is there a way to add a contour to the test we have right now?

Reply:
Yes, it's just a nibabel load. See `BBRegisterRPT`.